app/src/main/java/com/hedvig/app/feature/birthday/BirthdayActivity.kt

```diff
@@ -0,0 +1,70 @@
+package com.hedvig.app.feature.birthday
+
+import android.animation.Animator
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.hedvig.app.R
+import com.hedvig.app.feature.loggedin.ui.LoggedInActivity
+import com.hedvig.app.util.extensions.view.hide
+import com.hedvig.app.util.extensions.view.show
+import kotlinx.android.synthetic.main.activity_birthday.*
+import timber.log.Timber
+
+
+class BirthdayActivity : AppCompatActivity() {
+
+    private var animationDuration = 900
```

Comment:
These can be moved to a `companion object` and be made `const val` 🙂

Reply:
also you can suffix the value with `L` to make it a long in the first place, as it's only used as a long:

```suggestion
    private const val animationDuration = 900L
```